felix.go

```diff
@@ -466,6 +466,12 @@ func servePrometheusMetrics(configParams *config.Config) {
 				prometheus.Unregister(prometheus.NewProcessCollector(os.Getpid(), ""))
 			}
 		}
+		gaugeHost := prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
Feels a bit odd to use `GaugeVec` here when there's only one metric, can you do something like this instead: 
```
prometheus.NewGauge(prometheus.GaugeOpts{
    Name: "felix_host",
    Help: "Felix host.",
    ConstLabels: prometheus.Labels{...}, 
})
```